pennylane/_qubit_device.py

```diff
@@ -63,6 +63,7 @@ class QubitDevice(Device):
 
     # pylint: disable=too-many-public-methods
     _asarray = staticmethod(np.asarray)
+    observables = {"PauliX", "PauliY", "PauliZ", "Hadamard", "Hermitian"}
```

Comment:
Could "Identity" pop in here as well? :) E.g. the ``Forest`` plugin has observables defined so, it would just be for making sure that nothing breaks

Reply:
(This would also help out with removing the ``observables`` class attributes from the ``QVMDevice`` and ``QPUDevice``)

Reply:
```suggestion
    observables = {"PauliX", "PauliY", "PauliZ", "Hadamard", "Hermitian, "Identity"}
```
Good catch!